backend/experiments/LibExperimentalStdLib/LibCLI.fs

```diff
@@ -75,6 +75,51 @@ let fns : List<BuiltInFn> =
       previewable = Impure
       deprecated = NotDeprecated }
 
+    { name = fn "System" "getEnv" 0
+      typeParams = []
+      parameters = [ Param.make "varName" TString "" ]
+      returnType = TOption TString
+      description =
+        "Gets the value of the environment variable with the given <param varName> if it exists."
+      fn =
+        (function
+        | _, _, [ DString varName ] ->
+          let envValue = System.Environment.GetEnvironmentVariable(varName)
+          match envValue with
+          | null -> Ply(DOption None)
+          | _ -> Ply(DOption(Some(DString envValue)))
+        | _ -> incorrectArgs ())
+      sqlSpec = NotQueryable
+      previewable = Impure
+      deprecated = NotDeprecated }
+
+
+    { name = fn "System" "getEnvVars" 0
+      typeParams = []
+      parameters = []
+      returnType = TList(TTuple(TString, TString, []))
```

Comment:
Tuple list is good, but should this be a `Map<string,string>` instead?

Reply:
I thought we don't yet support Maps/Dicts. Should I instead return a `TDict`?

Reply:
Sorry, yeah we support TDict/DDict when the key is a string.

Reply:
I return a `TDict TString` now. We can't yet serialize Dicts, right? I've transformed the dict into a list of tuples before serializing and returning it through the http handler.

Reply:
We can't directly Json.parse and .serialize a Dict currently without also providing a Record type that matches the dict's shape, but I don't think we need to.

If we have the Map, and we extract the value of the API key, we're never trying to Json-serialize

In other words:
```fsharp
let openAiApiKey = Environment.getAll |> Map.get "OPENAI_API_KEY"
type ApiRequestBody = {  apiKey: string; ... }
let requestBody = { apiKey = openAiApiKey; ... } |> Json.serialize<ApiRequestBody> |> String.toBytes
...
```

Reply:
The reason I added the `getAll` was to see what variables are available to the canvas. Initially I only had the `get` and it didn't have access to `OPENAI_API_KEY` and I wasn't sure if that's an issue with my code or the environment.

Reply:
The reason that OPEN_API_KEY is not available is because env vars are loaded when the process starts (which in this case means when the server process starts). So env vars likely weren't a good fit for what you were trying to do. However, they're still useful for LibCLI.

Reply:
> I return a `TDict TString` now. We can't yet serialize Dicts, right?

Good point, fixing that at the moment.